### PR TITLE
Common details. Right sidebar logic added. Discussions and Proposals Api Calls added to common detail page.

### PR DESCRIPTION
**Pull request by razoorka**

### CODE REVIEW

File: packages/web/src/containers/Auth/store/reducer.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-import { tokenHandler } from "../../../shared/utils";
+// import { tokenHandler } from "../../../shared/utils";
 import * as actions from "./actions";
 import { ActionType, createReducer } from "typesafe-actions";
 
 type Action = ActionType<typeof actions>;
 
 const initialState: AuthStateType = {
-  authentificated: !!tokenHandler.get(),
+  authentificated: true,
```
Comment (alexander2001i) [inline packages/web/src/containers/Auth/store/reducer.tsx:11]: Why?

File: packages/web/src/containers/Common/containers/CommonDetailContainer/CommonDetailContainer.tsx [hunk 2]
```diff
@@ -1,26 +1,142 @@
...
   useEffect(() => {
     dispatch(getCommonDetail.request(id));
     return () => {
       dispatch(getCommonDetail.success(null));
     };
   }, [dispatch, id]);
+
+  const lastestProposals = useMemo(
+    () =>
+      [...proposals].splice(0, 5).map((p) => {
```
Comment (alexander2001i) [inline packages/web/src/containers/Common/containers/CommonDetailContainer/CommonDetailContainer.tsx:53]: What? Why do you need to spread than make back to array?

**Review by alexander2001i — APPROVED** [review]
Looks good to me. I just have few questions:

File: packages/web/src/containers/Auth/store/reducer.tsx (hunk 1 above)
Comment (razoorka) [reply packages/web/src/containers/Auth/store/reducer.tsx:11]: Cause, right now auth logic hasn't been described by the management team, and without authenticated APP will be redirected to auth page. It is a hack until auth will be described.

File: packages/web/src/containers/Common/containers/CommonDetailContainer/CommonDetailContainer.tsx (hunk 2 above)
Comment (razoorka) [reply packages/web/src/containers/Common/containers/CommonDetailContainer/CommonDetailContainer.tsx:53]: I have created a new array from base one with only 5 elements and different type. So, 1) I need only 5 elements from the whole pool of data. 2) I need another type of array of nested component.

**Pull request merged**